### ISSUE TITLE
VLC Player - Video Output

### ISSUE THREAD

**Issue by codekoch** [maxnet/berryboot#546]
The new built in VLC-Player version (Raspbian over Berryboot) is flickering and does not show any video output. 
I guess it's because of the kernel version 4.14.58v7-aufs with comes with berryboot.
In the new rasbian version with kernel 4.14.79-v7+ there is no problem with the VLC Player.
I tried to compile berryboot with the new kernel, but i had no sucess.
It ends up with an error....

**Comment by maxnet** [comment]
Try extracting the files from Berryboot version 20181128 to the FAT partition of the SD card.
It has kernel version 4.14.83

**Comment by codekoch** [comment]
Ok. Now I have a Raspbian with Kernel 4.14.83-aufs. 
But I get still the same problems with the VLC Player. It plays no video at all with Berryboot as Bootloader....Without Berryboot there is no problem. Any Idea? Can you confirm the problem?

**Comment by macmpi** [comment]
Maybe GPU needs more RAM for your usecase: you may want to try increasing gpu_mem in `config.txt` to 192, 448, or 944 depending on your Pi RAM configuration.

**Comment by maxnet** [comment]
Probably boot firmware version specific.
20181211 better?

**Comment by codekoch** [comment]
Higher gpu_mem settings don't solve the problem. I have also no special usecase...
I just downloaded the default raspbian full image, which is suggested by Berryboot and tried to start the vlc with several movies (avi, mkv etc.). 
@maxnet : I tried the new Berryboot build 20181211. And YES!!!! That solved my problem. But what is the difference? I thought I could update the firmware by rpi-update...

In any case, thank you all for the hints
